dev_guide/builds.adoc

```diff
@@ -1543,6 +1548,14 @@ webhook endpoint for your build:
 http://<openshift_api_host:port>/oapi/v1/namespaces/<namespace>/buildconfigs/<name>/webhooks/<secret>/generic
 ----
 
+The caller must invoke the webhook as a `*POST*` operation.
+
+To invoke the webhook manually you can use `*curl*`:
+
+    $ curl -X POST -k https://<openshift_api_host:port>/oapi/v1/namespaces/<namespace>/buildconfigs/<name>/webhooks/<secret>/generic
```

Comment:
As above, add ---- before and after the code snippet
(Per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)